internal/datastore/mention.go

```diff
@@ -0,0 +1,40 @@
+package datastore
+
+import (
+	"context"
+	"database/sql"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/nedrocks/delphisbe/graph/model"
+	"github.com/sirupsen/logrus"
+)
+
+func (d *delphisDB) PutActivity(ctx context.Context, tx *sql.Tx, post *model.Post) error {
+	logrus.Infof("PutActivity::SQL Create")
+	if err := d.initializeStatements(ctx); err != nil {
+		logrus.WithError(err).Error("PutActivity::failed to initialize statements")
+		return err
+	}
+
+	for _, entityID := range post.PostContent.MentionedEntities {
+		s := strings.Split(entityID, ":")
+
+		// Don't record mentions where a user tags themselves. This can also be handled on the frontend
```

Comment:
might be cleaner to handle on the frontend. 
we should also discuss what will be passed back when a user writes the token in their text and it isn't an actual token. this can be where the frontend passes back the location of the tokens